experimental/add_to_app/README.md

```diff
@@ -26,10 +41,15 @@ uses CocoaPods, so prior to running it for the first time, use this command
 to set up the workspace file:
 
 ```bash
-cd SimpleIOSExample
+cd example_module
```

Comment:
nit: Consider adding the step of cd-ing into the module and getting packages to the module section above (e.g. "Before running either of the projects below, you'll need to get packages for the Flutter module. To do that...")

Reply:
oh ya, good idea